docs/portal/image-registry/adding-projects.md

```diff
@@ -1,17 +1,19 @@
 # Adding Projects
 
 Admin portal users can use the following procedure to add projects.
+
 **Note:** Non-admin portal users are not permitted to add projects.
+
 1. In the left pane, click **Image Registry > Registry Projects**.
 2. At the top right of the Registry Projects page, click the **\+ Add New Project** button. 
 3. Complete the fields in the Add Project form. Required fields are denoted by an asterisk (\*).
 
-![null](</docs/resources/images/registry/add-project.png>)
+<p align=center><img src="/docs/resources/images/registry/add-project.png" width="600"></p>
```

Comment:
In previous documentations, screenshots are aligned to the left, and there isn't explicit image width specification. 

We'd better do the same for the image here and the images in other pages

Reply:
Fixed width and alignment for all figures. I kept the 600 width for most images. But if an image looked too big at 600 width, I reduced the size, but kept the alignment centered. Please let me know if you come across images that do not look sized properly. Also, please note that I need to engage Eleazar to capture high-resolution screen shots for 4 updated screens.

Reply:
I notice that some images have 700 width, e.g. the images on the "retrieving instances of an application" page. Is this intended?

Both 600 width and 700 width look fine to me. I raise this quesiton to you in case you do not intend images to have width larger than 600.